testing/acceptance_test.py

```diff
@@ -238,7 +239,7 @@ def test_issue88_initial_file_multinodes(self, pytester: Pytester) -> None:
         pytester.copy_example("issue88_initial_file_multinodes")
         p = pytester.makepyfile("def test_hello(): pass")
         result = pytester.runpytest(p, "--collect-only")
-        result.stdout.fnmatch_lines(["*MyFile*test_issue88*", "*Module*test_issue88*"])
+        result.stdout.fnmatch_lines(["*Module*test_issue88*", "*MyFile*test_issue88*"])
```

Comment:
this changes the meaning of the test - the myfile nodes are important

Reply:
Previous tree:

```
<MyFile test_issue88_initial_file_multinodes.py>
  <MyItem hello>
<Module test_issue88_initial_file_multinodes.py>
  <Function test_hello>
```

New tree:

```
<Dir test_issue88_initial_file_multinodes0>
  <Module test_issue88_initial_file_multinodes.py>
    <Function test_hello>
  <MyFile test_issue88_initial_file_multinodes.py>
    <MyItem hello>
```

I can't say exactly why the order changes but I can check if it's important. I'm not sure what issue88 is, I don't think it's github #88.

Reply:
The example code deals with custom collection

Reply:
The custom collector `MyFile` still works as before, just the collector/item order is changed. Do you think this test specifically checks for a regression in the order?

Reply:
The Test specifically Checks for the custom collection being applied,the change removes that

Reply:
The now changed Test would miss if the custom collector stopped working

Reply:
If the custom collector stops working, the `"*MyFile*test_issue88*"` won't match, and the test will fail, unless I'm missing something?

Reply:
apologies, i did misread the diff and thus missinterpreted the order of things wrongly